src/conditions/triggeredByBot.ts

```diff
@@ -9,7 +9,9 @@ export const BOT_USERS = [
   'dependabot-preview[bot]',
   'tf-security',
   'seti-tf',
-  'tf-IT'
+  'tf-seti',
```

Comment:
looks like this is duplicated, perhaps should we remove one of them?

Reply:
I simply removed that change now. I am not very sure that this actually filters all the tf-seti bots as it is now, as their author name I have seen it as "Engineering Intelligence", but I guess that's another thing to tackle (rename the bots for example).